config/jobs/kubernetes/sig-release/release-branch-jobs/1.23.yaml

```diff
@@ -1031,6 +1031,8 @@ presubmits:
       preset-pull-kubernetes-e2e-gce: "true"
       preset-service-account: "true"
     name: pull-kubernetes-e2e-gce-ubuntu
+    annotations:
```

Comment:
is this change intentional?

Reply:
yes it's needed, because L216 in other file created the group... we did the same in https://github.com/kubernetes/test-infra/pull/24911